src/match/serializers.py

```diff
@@ -0,0 +1,133 @@
+import json
+
+from location.serializers import LocationSerializer
+from match.models import Match
+from person.serializers import UserSerializer
+from rest_framework import serializers
+
+
+class MatchSerializer(serializers.ModelSerializer):
+    matched_user = serializers.SerializerMethodField("get_matched_user")
+    accepted_ids = serializers.SerializerMethodField("get_accepted_ids")
+    proposed_meeting_times = serializers.SerializerMethodField(
+        "get_proposed_meeting_times"
+    )
+    proposed_locations = serializers.SerializerMethodField("get_proposed_locations")
+    meeting_location = serializers.SerializerMethodField("get_meeting_location")
+
+    def __init__(self, *args, **kwargs):
+        self.request_user = kwargs.pop("user")
+        super().__init__(*args, **kwargs)
```

Comment:
wait why do you need this im very confused

this is just a serializer

Reply:
maybe im high

Reply:
and i dont actually know hwat im talking abt

Reply:
@alanna-zhou i added it to pass in the current user into the serializer so that the serializer returns the correct `matched_user` since frontend only wants us to return that